Makefile

```
@@ -93,7 +93,7 @@ test-all: lint test test-int
 
 deps:
 	$(info INFO: Starting build $@)
-	glide install
+	dep ensure
```

```suggestion
./dep ensure
```

(Only if you go with the previous suggestion though)

---

:+1: